samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,88 @@
+
+Requirements
+************
+
+The sample supports the following development kits for testing the network status:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set8_start
+   :end-before: set8_end
+
+Border Router higher layer process is needed to communicate with MCU border router part.
+Optionally, one or more compatible development kits programmed with OpenThread samples to test communication in bigger network.
+
+User interface
+**************
+
+Main recommended interaction with the application should be done via wpantund Linux process using its comannds (for more information check: `wpantund usage`_).
+It is also possible to communicate with board using pyspinel tool via serial interface (for more information check: `pyspinel commands`_).
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/openthread/ncp`
+
+|enable_thread_before_testing|
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+To perform tests it is necessary to have at least one development kit board and second Thread node for checking connection.
+Depending on needs it might be another development kit board running OpenThread stack or the simulated node on PC.
+
+Below steps were based on the given connectivity test: `OpenThread Border Router connectivity test`_, where detailed description may be found.
+
+#. Connect board to the PC and get its serial port name.
+#. Run OTBR using docker and pass board serial port name as ncp interface.
+#. Now bring up second Thread node, for example by flashing development kit board with OpenThread CLI FTD_JOINER sample (see here: `Codelabs - Build Thread network`_)
+#. Invoke some of the OpenThread commands to make new node to join to the border router network:
+
+  .. code-block:: console
+
+    uart:~$ dataset panid <network_panid>
+    uart:~$ dataset masterkey <network_masterkey>
+    uart:~$ dataset commit active
+    uart:~$ ifconfig up
+    uart:~$ thread start
+
+    Parameters: network_panid and network_masterkey can be read from the OTBR process output logs
+#. At this point, typing "state" command into the node CLI interface should end with answer "child" or "router" depending on running time.
+#. Invoke some external IPv6 address 16 bytes from 64:ff9b:0:0:0:0:acd9:a46e: icmp_seq=1 hlim=118 time=45mss ping command to prove that it is possible to communicate with hosts from outside of our network:
+
+  .. code-block:: console
+
+    uart:~$ ping 64:ff9b::acd9:a46e
+    16 bytes from 64:ff9b:0:0:0:0:acd9:a46e: icmp_seq=1 hlim=118 time=45ms
+
+
+Dependencies
+************
+
+This sample uses the following Zephyr libraries:
+
+* :ref:`zephyr:kernel_api`:
+
+  * ``include/kernel.h``
+
+* :ref:`zephyr:thread_protocol_interface`
+
+* :ref:`zephyr:logging_api`:
+
+  * ``include/logging/log.h``
```

Comment:
```suggestion
  * ``include/logging/log.h``

----

Copyright disclaimer
    |Google_CCLicense|

```
You need an extra blank line at the end, and the copyright disclaimer, as you're referring to the OpenThread user guide.

Reply:
Done